### ISSUE TITLE
go.mod: upgrade dependency golang.org/x/crypto to latest version to avoid vulnerability

### ISSUE THREAD

**Issue by mswathi-tw** [segmentio/kafka-go#944]
We are using the latest version (github.com/segmentio/kafka-go@v0.4.32) which internally uses the following library

golang.org/x/crypto v0.0.0-20190506204251-e1dfcc566284 

This has introduced certain snyk vulnerability **Improper Signature Verification** and in order to solve this issue following library version has to upgraded to golang.org/x/crypto@0.0.0-20200220183623-bac4c82f6975 or more.

Can you please upgrade it.

**Comment by achille-roussel** [comment]
Hello @mswathi-tw 

Are you available to send a pull request to update the dependency?

**Comment by mswathi-tw** [comment]
Hi @achille-roussel , I do not have enough access to create a PR
ERROR: Permission to segmentio/kafka-go.git denied to mswathi-tw.

**Comment by achille-roussel** [comment]
Yes, the workflow would be to fork the repository, make the change in your fork and submit a pull request from it 👍

**Comment by mswathi-tw** [comment]
I have created a PR to update all dependencies: https://github.com/segmentio/kafka-go/pull/945/files

**Comment by achille-roussel** [comment]
Thanks! I merged it in!
